guides/creating-a-custom-form-field-control.md

```diff
@@ -24,11 +24,11 @@ class MyTel {
   selector: 'example-tel-input',
   template: `
     <div [formGroup]="parts">
-      <input class="area" formControlName="area" size="3">
+      <input class="area" formControlName="area" size="3" maxlength="3">
```

Comment:
I think that we can drop the `size` here as well. According to MDN it's only supported for `email`, `password`, `tel` inputs.

Reply:
Agree!